libsolidity/formal/SMTLib2Interface.h

```diff
@@ -67,10 +71,12 @@ class SMTLib2Interface: public SolverInterface, public boost::noncopyable
 	/// Communicates with the solver via the callback. Throws SMTSolverError on error.
 	std::string querySolver(std::string const& _input);
 
-	ReadCallback::Callback m_queryCallback;
+	std::map<h256, std::string> const& m_smtlib2Responses;
 	std::vector<std::string> m_accumulatedOutput;
 	std::set<std::string> m_constants;
 	std::set<std::string> m_functions;
+
+	std::vector<std::string> m_unhandledQueries;
```

Comment:
I guess we have to name this like `m_smtlib2UnhandledQueries`

Reply:
Never mind, this is in `SMTLib2Interface. In this case then we could rename `m_stlib2Responses` to `m_externalResponses` or just `m_queryResponses`?

Reply:
`m_smtlib2Responses` is above, this is something else, right?

Reply:
It is just confusing that these exists on `CompilerStack` and here with inconsistent names.

For this file I'd suggest to group the two members together and name them:
- `m_queryResponses`
- `m_unhandledQueries` or `m_queryRequests`